moveit_kinematics/CMakeLists.txt

```diff
@@ -33,21 +50,23 @@ pluginlib_export_plugin_description_file(moveit_core cached_ik_kinematics_plugin
 
 include_directories(${THIS_PACKAGE_INCLUDE_DIRS})
 
+add_subdirectory(cached_ik_kinematics_plugin)
+add_subdirectory(ikfast_kinematics_plugin)
 add_subdirectory(kdl_kinematics_plugin)
 add_subdirectory(lma_kinematics_plugin)
 add_subdirectory(srv_kinematics_plugin)
-add_subdirectory(ikfast_kinematics_plugin)
-add_subdirectory(cached_ik_kinematics_plugin)
 add_subdirectory(test)
 
-ament_export_libraries(
```

Comment:
Shouldn't the `moveit_cached_ik_kinematic_base` plugin still be exported if not the others?

Reply:
all right, self-answered